clients/privacy-center/public/fides-js-demo.html

```diff
@@ -4,8 +4,19 @@
     <title>fides-js script demo page</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <script src="/fides.js"></script>
-
+    <!-- 
+      Pass along any params to the fides.js script. For example, visiting
+      http://localhost:3000/fides-js-demo.html?geolocation=fr-id
+      will pass a geolocation query param to fides.js
+    -->
+    <script>
+      (function () {
+        const params = window.location.search;
+        const src = `/fides.js${params ?? ""}`;
+
+        document.write('<script src="' + src + '"><\/script>');
```

Comment:
I think this should be ok since it's a test page but maybe I'm not being paranoid enough!

Reply:
Hm, since this is a public-facing page, I think we should treat this as legit and be extra careful. I'm unable to perform a XSS attack manually, but I think we should at least use our `VALID_ISO_3166_LOCATION_REGEX` regex here.

Reply:
good point, let me try that!